preupg/settings.py

```diff
@@ -43,6 +43,15 @@
 cache_dir = "/var/cache/preupgrade"
 log_dir = "/var/log/preupgrade"
 
+# file with meta info
+properties_ini = "properties.ini"
+# section inside ini file
+section_name = "preupgrade-assistant-modules"
+# key which specify source major OS version
+src_version_key ="srcMajorVersion"
+# key to specify destination major OS version
+dest_version_key ="destMajorVersion"
```

Comment:
Expected behaviour after talking to @pirat89:

- When you encounter _properties.ini_ in the place where it is expected (_/usr/share/preupgrade/<module_set_dir>/properties.ini_) and it has no "preupgrade-assistant-modules" section print out warning, something like _"Found 'properties.ini' in \<dir\>. If it's supposed to be ini file for a module set, it needs to contain 'preupgrade-assistant-modules' section."_

- When the _properties.ini_ file has _"preupgrade-assistant-modules"_  section but it has either of the two options missing (_"srcMajorVersion"_ or _"destMajorVersion"_) print out an error message and stop Preupgrade Assistant execution.